### ISSUE TITLE
Question about prediction_scenes.json

### ISSUE THREAD

**Issue by peiyunh** [nutonomy/nuscenes-devkit#312]
Hi, 

It looks like there are more observations in both the train set and the validation set in [prediction_scenes.json](https://github.com/nutonomy/nuscenes-devkit/blob/nuscenes-predict-challenge/python-sdk/nuscenes/eval/predict/prediction_scenes.json) than what is mentioned in the [CoverNet](https://arxiv.org/pdf/1911.10298.pdf) paper. 

Quoting the paper:
> This leaves us with 37,714 observations in the train set, 8,064 observations in the validation set, and 7,790 observations in the test set. 

In `prediction_scenes.json`, 
```
>>> len(splits.get_prediction_challenge_split("train"))
109794
>>> len(splits.get_prediction_challenge_split("val"))
21630
```

I am guessing stationary observations have not been filtered out of `prediction_scenes.json`?

Thanks,
Peiyun

**Comment by peiyunh** [comment]
I checked the ground truth future trajectories. It looks like none of them are really stationary. 

The initial reason I ask about it is that I cannot reproduce the numbers reported in the CoverNet paper in terms of the performance of baselines, including the constant-velocity model and the physics oracle. Below, I report the numbers I get and include the numbers reported by the paper in parentheses. 

For the constant-velocity model's validation performance, 
```
MinADEK = 5.32 (vs. 3.75)
MinFDEK = 12.9 (vs. 9.44)
HitRateTopK_2 = 0.23 (vs. 0.19)
```

For the physics oracle model's performance, I have
```
MinADEK = 3.13 (vs. 2.79)
MinFDEK = 6.96 (vs. 7.18) 
HitRateTopK_2 = 0.38 (vs. 0.23)
```

Is the reason why we see different numbers that `prediction_scenes.json` contains a different set of observations than what was used for the paper? 

Thanks.

**Comment by freddyaboulton** [comment]
Hi @peiyunh! Thanks for the interest. This branch is currently under active development so the code will not have been reviewed until it is on master. We plan on merging in the next few weeks.

**Comment by peiyunh** [comment]
Got it. Please feel free to close it!
